src/Microsoft.DotNet.Build.Tasks/PackageFiles/Build.Common.props

```diff
@@ -23,6 +23,20 @@
 
   <!-- Common BuildTools properties -->
   <PropertyGroup>
+    <!--
+        Build Configuration is composed by: $(OSGroup).$(Platform).$(ConfigurationGroup).$(TargetGroup)
+        TargetGroup will be "default" if no value is specified. 
+    -->
+    <BuildConfiguration>$(OSGroup).$(Platform).$(ConfigurationGroup)</BuildConfiguration>
```

Comment:
BuildConfiguration will end up being the property but it will end up conflicting with your changes here. To help avoid this conflict can you please put the definition of this inside its own PropertyGroup and condition it on BuildConfiguration not already being set?